lesson06/script.js

```diff
@@ -0,0 +1,25 @@
+const lists = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+];
+
+const promise = new Promise(resolve => {
+    setTimeout(() => resolve(lists),3000);
+});
```

Comment:
I implemented using setTimeout.
I learned that the function specified by setTimeout is called only once.